challenges/objects-arrays.js

```diff
@@ -7,28 +7,64 @@
 */
 
 // tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceous
-
+function weight() {
+  console.log(this.weight)
+}
+
+function rawr(){
+  console.log(this.rawr);
+}
+
+let tyrannosaurus = {
+  name: 'tyrannosaurus',
+  diet: 'carnivorous',
+  weight: '7000kg',
+  length: '12m',
+  period: 'Late Cretaceous',
+  rawr: 'RAWERSRARARWERSARARARRRR!',
+  live: function(){ 
+    console.log(this.period);
+  }
+}
 // stegosaurus, herbivorous, 2000kg, 9m, Late Jurassic
-
+let stegosaurus = {
+  name: 'stegasaurus',
+  diet: 'herbivorous',
+  weight: '2000kg',
+  length: '9m',
+  period: 'Late Jurassic',
+  food: function() {
+    console.log(this.diet);
+  }
+}
 // velociraptor, carnivorous, 15kg, 1.8m, Late Cretaceous
-
+let velociraptor = {
+  name: 'velociraptor',
+  diet: 'carnivorous',
+  weight: '15kg',
+  length: '1.8m',
+  period: 'Late Cretaceous',
+  long: function() {
+    console.log(this.length);
+  }
+}
 // Using your dinosaur objects, log answers to these questions:
 
 // How much did tyrannosaurus weigh?
-console.log();
+weight.call(tyrannosaurus);
```

Comment:
The way you set up this code, was by creating functions that `console.log()` certain properties. But, you could also write something like this
```
console.log(tyrannosaurus.weight)
```
And it would output the same thing. Both your code and the code typed here produce the same output of "7000kg"; however, your code requires an extra function(`weight()`) that the code here doesn't need